inspirehep/dojson/utils/__init__.py

```diff
@@ -224,3 +227,8 @@ def dedupe_all_lists(obj):
         return type(obj)(new_obj)
     else:
         return obj
+
+
+def validate(instance, schema):
+    return jsonschema_validate(
+        instance, schema, resolver=LocalRefResolver('', {}))
```

Comment:
@david-caro was telling that the one responsible for the validation of the JSON Schema should be the `inspire_schema`. Probably we should move these lines in this method https://github.com/inspirehep/inspire-schemas/blob/master/inspire_schemas/api.py#L32 in according to the behavior that we need. What do you think?